src/components/spectrogram/spectrogram.ts

```diff
@@ -160,6 +170,9 @@ export class SpectrogramComponent extends SignalWatcher(ChromeHost(LitElement))
   private readonly highAccuracyTimeBuffer = new SharedArrayBuffer(Float32Array.BYTES_PER_ELEMENT);
   private readonly currentTimeBuffer = new Float32Array(this.highAccuracyTimeBuffer);
 
+  // TODO: move somewhere else
+  private timeInterpolateFrameId: number | null = null;
```

Comment:
better name? this is a RAF reference yeah?

Reply:
Do you have any suggestions?

- `timeFrameRequestId`?
- `timeInterpolateFrame`? (without `Id`)
-

Reply:
We settled on `interpolationCancelReference`